BlackJackGame/Deck.cs

```diff
@@ -48,37 +48,37 @@ public List<string> GenerateDeck()
                             break;
                     }*/
                     
-                    string card = _suites[i] + _values[j];
-                    _completeDeck.Add(card);
+                    string card = values[j] + " " +  suites[i];
+                    completeDeck.Add(card);
                 }
             }
             Console.WriteLine("\ncompleted deck:");
-            for (int i = 0; i <_completeDeck.Count; i++)
+            for (int i = 0; i <completeDeck.Count; i++)
             {
-                Console.WriteLine(_completeDeck[i]);
+                Console.WriteLine(completeDeck[i]);
             }
             //completeDeck.ForEach(Console.WriteLine);
             //Console.Write(completeDeck.Count);
             //Shuffle(completeDeck);
 
-            return _completeDeck;
+            return completeDeck;
         }
 
         public List<string> Shuffle(List<string> completeDeck)
```

Comment:
When you create an instance of `Deck` in your `Game` class, that object of `Deck` represents the game deck.
```
var deck = new Deck()
```
So `deck` variable is basically representing your deck of cards. Soo when you call shuffle like
```
deck.shuffle(anotherdeck)
```
Its really weird, because you are passive ANOTHER deck into your deck. If you read my above comment about making your ```_completeDeck``` an class variable, its not accessible by the whole class including the shuffle method, so you can instead write it as.
```
deck.Shuffle()
```